hoot-core/src/main/cpp/hoot/core/io/OsmApiChangeset.cpp

```diff
@@ -995,6 +1018,45 @@ ChangesetInfoPtr XmlChangeset::splitChangeset(ChangesetInfoPtr changeset, const
         }
       }
     }
+    //  See if the hint is something like: Changeset precondition failed: Precondition failed: Node 5 is still used by ways 67
```

Comment:
This is the message and here is the code to try to handle it, isolating the failing element and trying to add the blocking element to the same changeset so that it will be successful next time.